3drepo/3drepo.io
# Issue 3584 - Ticket List Item Components

This fixes #3584 

#### Description
- Adds the components for the ticket list item. This includes:
    - Assignees list
    - Filter chip
    - Status chip
    - Priority chip
    - Treatment chip
    - Risk chip
    - Due date
- Add these new components to the ticket list item
- This included updating some icons (Bell icon)

#### Test cases
- Add some new tickets with varying properties. (Test with both null values and by not defining the values. Note that the null state for assignees is an empty array)
- View the tickets list
